core/src/main/scala/akka/persistence/jdbc/journal/dao/JournalQueries.scala

```diff
@@ -69,6 +69,15 @@ class JournalQueries(
     JournalTable.filter(_.persistenceId === persistenceId).filter(_.sequenceNumber <= toSequenceNr).delete
   }
 
+  private[akka] def markAsDeleted(persistenceId: String, seqNr: Long) =
+    JournalTable
+      .filter(_.persistenceId === persistenceId)
+      .filter(_.sequenceNumber === seqNr)
+      .filter(_.deleted === false)
+      .map(_.deleted)
+      .update(true)
```

Comment:
My Slick knowledge is rusty, does the `.map(_.deleted)` somehow change the boolean value of the field `deleted` or what does this do?

Reply:
Thanks for clarifying